python/ray/widgets/util.py

```diff
@@ -102,6 +100,8 @@ def _has_outdated(
     outdated = []
     for (lib, version) in deps:
         try:
+            from packaging.version import Version
```

Comment:
is it necessary?

Reply:
Yes, somehow it won't work without moving it. Complaints about importing issues. not sure why.